Themes/DarkMode.scss

```diff
@@ -0,0 +1,177 @@
+// OMNIA Low-Code
+// Dark Theme 1.0
+
+//
+// Color system
+//
+
+$white:    #fff !default;
```

Comment:
remove the !default tag from all variables.

Reply:
Done